packages/web/src/pages/audio-rewards-page/components/modals/ChallengeRewards.tsx

```diff
@@ -245,7 +246,7 @@ const ChallengeRewardsBody = ({ dismissModal }: BodyProps) => {
   const progressReward = (
     <div className={wm(styles.progressReward)}>
       <h3>Reward</h3>
-      <h2>{challenge?.totalAmount}</h2>
+      <h2>{formatNumberCommas(challenge?.totalAmount || '')}</h2>
```

Comment:
this fn isn't typed, but i think it takes a num instead of a string, so the empty string is maybe not the right undefined default (though it may work).

perhaps the fn itself could just take undefined and if undefined return ''

Reply:
It takes either - if the arg is a number, it calls `toString()` first anyway.

I think it makes sense to have `Earn up to  $AUDIO` in the worst case rather than `Earn up to 0 $AUDIO`?

Reply:
don't get me wrong, I don't think a default of 0 is correct, but since the arg is called num, I think this is confusing. We should probably just type the util haha.

Reply:
It has JSDoc, which TypeScript used to warn me that the arg isn't allowed to be undefined. I didn't find it confusing fwiw. also just realizing this should be ?? '' not || '' probably